UITests/SecurityTests.swift

```diff
@@ -49,11 +49,15 @@ class SecurityTests: KIFTestCase {
     /// in a new tab via window.open(). Make sure nothing happens.
     func testErrorExploit() {
         // We should only have one tab open.
-         tester().waitForTappableView(withAccessibilityLabel: "Show Tabs", value: "2", traits: UIAccessibilityTraitButton)
+        var value = tester().waitForView(withAccessibilityIdentifier: "TabToolbar.tabsButton")
+        let tabcount = value?.accessibilityValue
+
         tester().tapWebViewElementWithAccessibilityLabel("Error exploit")
-        
+
         // Make sure a new tab wasn't opened.
-        tester().waitForTappableView(withAccessibilityLabel: "Show Tabs", value: "2", traits: UIAccessibilityTraitButton)
+        value = tester().waitForView(withAccessibilityIdentifier: "TabToolbar.tabsButton")
+        let newTabcount = value?.accessibilityValue
+        XCTAssertEqual(tabcount, newTabcount)
```

Comment:
` XCTAssert(tabcount > 0 && tabcount == newTabcount) `
Since if there is an error in getting the value it will be 0 and this will detect that.